ui-tests/tests/ui.spec.ts

```diff
@@ -7,14 +7,10 @@ test.describe('UI Test', () => {
   const fileList = ['test.jcad', '3M_CONNECTOR.STEP', 'fan.stl'];
 
   test.describe('Extension activation test', () => {
-    test('should emit an activation console message', async ({
-      page,
-      request
-    }) => {
+    test('should emit an activation console message', async ({ page }) => {
       const logs: string[] = [];
 
       page.on('console', message => {
-        console.log('CONSOLE MSG', message.text());
```

Comment:
Shouldn't we keep this? It's useful to get the logs in the terminal that runs the test

Reply:
Yes, it was removed while debugging and pushed by mistake, will revert